src/hypertable.c

```diff
@@ -2046,6 +2047,13 @@ ts_hypertable_create(PG_FUNCTION_ARGS)
 Datum
 ts_hypertable_distributed_create(PG_FUNCTION_ARGS)
 {
+#if PG16_GE
+	ereport(ERROR,
+			(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
+			 errmsg("distributed hypertable is not supported"),
+			 errhint("Multi-node is not supported anymore on PostgreSQL >= 16."),
+			 errdetail("Use the \"create_hypertable\" to create a regular hypertable instead.")));
+#endif
```

Comment:
Isn't this effectively an API-breaking change, which we should not introduce in a minor release? Features can be deprecated in minor releases, but removing support for them does not sound like a good idea. Also, we deprecate it for all PG versions, not only PG16.

```suggestion
	ereport(WARNING,
			(errcode(ERRCODE_WARNING_DEPRECATED_FEATURE),
			 errmsg("distributed hypertables are deprecated"),
			 errhint("Distributed hypertables are deprecated and will be removed in the future.")));
```

Reply:
Fixed!